native/src/main/resources/error.properties

```diff
@@ -70,3 +70,6 @@ error.xml.parse.error=\
 
 error.undefined.field=\
   undefined field ''{0}'' in record ''{1}''
+
+error.attributes.cannot.convert.to.array.type=\
```

Comment:
Can't we have a more common error message like, 
`attribute ''{0}'' cannot be converted into  type ''{1}''` this so we can reuse this for non-array types as well

Reply:
in here IMOl we should enforce the `array` word.
Because attributes cannot have arrays